pkg/virt-handler/vm_test.go

```diff
@@ -417,14 +420,18 @@ var _ = Describe("VirtualMachineInstance", func() {
 		It("should attempt graceful shutdown of Domain if no cluster wide equivalent exists", func() {
 			vmi := v1.NewMinimalVMI("testvmi")
 			vmi.UID = vmiTestUUID
+
```

Comment:
This test is now failing, but I'm not actually sure which case it is trying to trigger / test. Any ideas?